Changes

```diff
@@ -9,6 +9,12 @@ Fixes
 - GraphComponent : Fixed bug that allowed construction with an invalid name (#3436).
 - Layouts : Keyboard shortcuts will now work in detached panels restored with a layout.
 - Rotate/TranslateTool : Fixed a potential crash if targeted mode was used with an empty selection.
+- TabbedContainer : Fixed a bug that caused an exception when the last tab in a container was closed.
+
+Breaking Changes
+----------------
+
+- TabbedContainer : 0.55.0.0 fixed an issue where incorrect widgets would be provided to `currentChangedSignal`. This however created a subsequent issue where exception was raised when the last tab was removed as there was no widget to pass to the signal. This is fixed in this release, but consequently, consumers of `currentChangedSignal` should handle the supplied widget being `None` when there are no widgets remaining in the container.
```

Comment:
We can't have breaking changes in a patch release. But I think the right approach here is to just classify this as a bugfix and remove this line. LGTM otherwise.

Reply:
Grand, better to get it out... removed in https://github.com/GafferHQ/gaffer/commit/d8c1f7e2ab0ba8c0765a7e7385b2483b2dd3601b.